### PR TITLE
fix: app crashed on lockdown

### PR DESCRIPTION
**Pull request by samsiegart**
The app was crashing on load when running locally (both with `yarn dev` and `yarn build`&`yarn preview`).
```
chunk-DRHPO5BB.js?v=8f44a6fd:2987 Error: Cannot initialize @agoric/assert, missing globalThis.assert methods Fail
  at assert.js:48:9
```
I'm not sure exactly what caused this to start happening, but updating the deps fixed it.

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -1,6 +1,4 @@
 import './installSesLockdown';
-import 'ses';
```
Comment (turadg) [inline src/main.tsx:2]: why is this necessary? did something change?

File: yarn.lock [hunk 2]
```diff
@@ -925,6 +925,11 @@
   resolved "https://registry.yarnpkg.com/@endo/base64/-/base64-0.2.27.tgz#6c68fe83092c621e2d46f77523e0e898e7f7fa68"
   integrity sha512-5LVFV7EDZemO+G8plo3pENFNmuCbo4G3Mnj8meRh9P7VyYlRK3QSvUe7QZ59BUb88iByPd6h0/a2PWyUC+zucw==
 
+"@endo/base64@^0.2.29":
```
Comment (turadg) [inline yarn.lock:928]: there are two versions in the lockfile now. we should have just one. `npx yarn-deduplicate` might do the job. If not we may need to bump all `@endo` deps to latest

File: src/main.tsx (hunk 1 above)
Comment (samsiegart) [reply src/main.tsx:2]: `installSesLockdown` already imports `ses` so it was redundant

File: yarn.lock (hunk 2 above)
Comment (samsiegart) [reply yarn.lock]: I think that did the trick, PTAL

File: src/main.tsx (hunk 1 above)
Comment (turadg) [reply src/main.tsx:2]: Very well. I wonder if we should even import `ses` there.  This looks like Endo's general lockdown handler: https://github.com/endojs/endo/blob/master/packages/lockdown/pre.js

Then there'd be less magic, because `lockdown` would come from an import instead of as a side effect.
```js
import { lockdown } from '@endo/lockdown';
```

@kriskowal is that the current best practice?

**Review by turadg — APPROVED** [review]

File: src/main.tsx (hunk 1 above)
Comment (samsiegart) [reply src/main.tsx:2]: Oh yea that works
